core/src/main/java/com/digitalpebble/stormcrawler/protocol/HttpRobotRulesParser.java

```diff
@@ -53,6 +55,12 @@ public void setConf(Config conf) {
         super.setConf(conf);
         allowForbidden = ConfUtils.getBoolean(conf, "http.robots.403.allow",
                 true);
+        fetchRobotsMd = new Metadata();
+        /* http.content.limit for fetching the robots.txt */
+        int robotsTxtContentLimit = ConfUtils.getInt(conf,
+                "http.robots.content.limit", 524288);
```

Comment:
default value should match what we have for http.content.limit and the default conf file => -1